### PR TITLE
Windows CI

### PR DESCRIPTION
**Pull request by isuruf**
`dump-file` tests fail. cc @saraedum 

https://ci.appveyor.com/project/isuruf/arb/builds/32420989

### CODE REVIEW

**Comment by isuruf** [comment]
@fredrik-johansson, this is still a WIP, but let me know what you think. You'll have to sign up for appveyor CI to set this up.

**Comment by fredrik-johansson** [comment]
Can you provide more information about the dump-file failure?

**Comment by fredrik-johansson** [comment]
Is it possible that tmpfile() doesn't work?

**Comment by isuruf** [comment]
Here's all the information that I have.
```
The following tests FAILED:
	388 - arb-t-dump_file (Exit code 0xc0000409
)
	660 - arf-t-dump_file (Exit code 0xc0000409
)
	767 - mag-t-dump_file (Exit code 0xc0000409
)
```

File: arb/test/t-dump_file.c [hunk 1]
```diff
@@ -33,6 +33,10 @@ int main()
         arb_randtest_special(x, state, 1 + n_randint(state, 1000), 1 + n_randint(state, 100));
 
         tmp = tmpfile();
+        if (tmp == NULL) {
+            flint_printf("FAIL (creating temporary file)  iter = %wd\n\n", iter);
+            flint_abort();
+        }
         arb_dump_file(tmp, x);
         fflush(tmp);
         rewind(tmp);
```
Comment (isuruf) [inline arb/test/t-dump_file.c:42]: The test segfaults `arb_load_file` on the line.

**Comment by fredrik-johansson** [comment]
Does the latest flint fix this issue by any chance?

**Comment by isuruf** [comment]
No, it doesn't.

**Comment by isuruf** [comment]
@fredrik-johansson, can you merge this before the next arb release which I understand is going to happen soon?

**Comment by fredrik-johansson** [comment]
Sure. What else needs to be done besides merging?

**Comment by isuruf** [comment]
Sign up at https://ci.appveyor.com and add arb as a project. Let me know if you run into any trouble.

**Comment by fredrik-johansson** [comment]
Okay, that's done. What's next?

**Comment by isuruf** [comment]
Thanks. You should see a add project at https://ci.appveyor.com/account/fredrik-johansson/projects. You can then add arb as a project.

**Comment by fredrik-johansson** [comment]
Seems to be working now :-)

**Pull request merged**